docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "from qiskit_experiments.test.t2hahn_backend import T2HahnBackend\n",
+    "# FakeJob is a wrapper for the backend, to give it the form of a job\n",
+    "from qiskit_experiments.test.utils import FakeJob\n",
+    "\n",
+    "\n",
+    "estimated_t2hahn2 = 20\n",
+    "# The behavior of the backend is determined by the following parameters\n",
+    "backend2 = T2HahnBackend(\n",
+    "    t2hahn=[20],\n",
+    "    frequency=[100100],\n",
+    "    initialization_error=[0.0],\n",
+    "    readout0to1=[0.02],\n",
+    "    readout1to0=[0.02],\n",
+    "    conversion_factor=conversion_factor,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "dt_factor2 = apply_prefix(1, unit2)\n",
+    "\n",
+    "# exp2.set_analysis_options(p0={\"amp\": 0.5, \"tau\": 20 * conversion_factor, \"base\": 0.5})\n",
+    "\n",
+    "expdata2 = exp2.run(backend=backend2, shots=2000)\n",
+    "expdata2.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "display(expdata2.figure(0))"
+   ]
+  },
```

Comment:
Throughout there is no explanation about the frequency parameter in the simulator. Explain it, and show that with one echo you get more accurate results than with zero echoes, when the frequency is not accurate.

Reply:
added under `Number of echoes` the following text:
`...  inaccuracy estimation. Due to that, the Hahn Echo experiment improve our estimate for  𝑇2 . In the following code, we will compare results of the Hahn experiment with 0 echoes and 4 echoes. In addition, We will add frequency to the qubit and see how the result change due to that (We can see Rabi Oscillations in the 0 echoes case) Note, that the provided delay time is the for each delay in the circuit and not the total time.`